static/js/pages/SiteCreationPage.test.tsx

```diff
@@ -0,0 +1,167 @@
+const mockUseRouteMatch = jest.fn()
```

Comment:
This is just what used to be `SitesDashboard.test.tsx` but renamed and with other variables renamed like `Website` => `WebsiteDetail`